python/tskit/tables.py

```diff
@@ -39,12 +39,27 @@
 attr_options = {"slots": True, "frozen": True, "auto_attribs": True}
 
 
-@attr.s(**attr_options)
+@attr.s(cmp=False, **attr_options)
```

Comment:
Hmm, better to use ``eq=False`` then, since ``cmp`` is being deprecated?

Reply:
You'd think so, but on my machine I have attrs 19.1.0, in which the `eq` parameter does not exist.

Reply:
I see. So cmp will be deprecated and we'll have to deal with that at some point. Maybe add a quick comment to that effect?

Reply:
done